packages/twenty-front/src/modules/client-config/types/ClientConfig.ts

```diff
@@ -30,7 +30,7 @@ export type ClientConfig = {
   isMicrosoftCalendarEnabled: boolean;
   isMicrosoftMessagingEnabled: boolean;
   isMultiWorkspaceEnabled: boolean;
-  isIMAPMessagingEnabled: boolean;
+  isImapSmtpCaldavEnabled: boolean;
```

Comment:
This is not how we usually do it for feature flags, feature flags are already passed to the frontend (see useIsFeatureEnabled). The client config booleans here serve another use-case (people who want to disable microsoft or google login on their server, but disabling imap on their server doesn't make as much sense and wouldn't be managed through flags anyway which are temporary)

Reply:
Aaah ok my bad so it's both a feature flag and a server-flag. OK, could you chose two slightly different names at least so that we can make the difference and ctrl+f to remove the feature flag later?